py3status/modules/wifi.py

```diff
@@ -89,15 +89,13 @@ def post_config_hook(self):
             raise Exception(STRING_NOT_INSTALLED)
 
         # get wireless interface
-        try:
+        if not self.device:
             data = self.py3.command_output([iw, "dev"])
-            devices = re.findall(r"Interface\s*([^\s]+)", data)
-            if not devices or "wlan0" in devices:
-                self.device = "wlan0"
-            else:
-                self.device = devices[0]
-        except self.py3.CommandError:
```

Comment:
What happens now if `CommandError` is thrown? Just simulate somehow. I think previously we tried to catch the errors, now we will produce module failure?

Reply:
It would break there. My PoV is that this `try/except` might be old code usually used with `subprocess.check_output`, a06882dac8602ea17fa2ea0e5f0518491ddc99ba, but we really should confirm this on Debian first.

`iw dev` say this does `List all network interfaces for wireless hardware.` so I think it would print nothing and not throwing errors. Yes on module failure if it we got one.